packages/main/src/RPA/Robocorp/WorkItems.py

```diff
@@ -288,21 +289,27 @@ class FileAdapter(BaseAdapter):
 
     Reads inputs from the given database file, and writes
     all created output items into an adjacent file
-    with the suffix ``<filename>.output.json``.
+    with the suffix ``<filename>.output.json``. If the output path is provided by an
+    env var explicitly, then the file will be saved with the provided path and name.
 
     Reads and writes all work item files from/to the same parent
     folder as the given input database.
 
     Required environment variables:
 
-    * RPA_WORKITEMS_PATH:   Path to work items database file
+    * RPA_INPUT_WORKITEM_PATH:   Path to work items database file
     """
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        path = required_env("RPA_WORKITEMS_PATH")
+        old_path = os.getenv("RPA_WORKITEMS_PATH", UNDEFINED_VAR)
+        if old_path is not UNDEFINED_VAR:
+            deprecation("Work items load - Old path style usage detected")
```

Comment:
Don't forget to put here the links to docs. (so user knows how to use the new style)

Reply:
Also in general I think the error can be a bit more verbose, and include the name of the env var in question

Reply:
Done, this is displayed like so now:
<img width="1424" alt="Screenshot 2021-09-27 at 12 19 50" src="https://user-images.githubusercontent.com/709053/134881274-22801357-0914-4860-91df-ee36f723dbe6.png">

----

The main advantage of it is that's displayed only once per line, so it comes handy when having a loop that would display N times the very same warning. (avoiding some logging spam)